ee/components/stripe/PaymentPage.tsx

```diff
@@ -42,6 +43,14 @@ const PaymentPage: FC<PaymentPageProps> = (props) => {
   }, []);
 
   const eventName = props.booking.title;
+  // TODO: Move to translations
+  const locationLabels = {
```

Comment:
Pushed up this code pending string confirmation. This code comes from
L123 of components/booking/pages/BookingPage.tsx